### PR TITLE
DO NOT MERGE: 269/Card style in main, about and gde activity

### PR DESCRIPTION
**Pull request by friedger**
More on cards in About and GDE

This includes also a small fix for the main activity.

Builds on top of #415.

device-2015-07-26-163612

### CODE REVIEW

File: app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java [hunk 1]
```diff
@@ -36,13 +36,13 @@
...
     public ContributorAdapter(Context context) {
-        super(context, R.layout.contributor_item);
+        super(context, R.layout.list_contributor_item);
     }
 
     @Override
     public View getView(int position, View convertView, ViewGroup parent) {
         if (convertView == null) {
-            convertView = View.inflate(getContext(), R.layout.contributor_item, null);
+            convertView = View.inflate(getContext(), R.layout.list_contributor_item, null);
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java]: Not this pr s scope but this should use parent, false pattern.

Comment (friedger) [inline app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java]: View.inflate does not have such a method. It is only the `LayoutInflator` I
think.


  [image: --]
Friedger Müffke
[image: http://]about.me/friedger
<http://about.me/friedger>


2015-07-26 17:51 GMT+02:00 Said Tahsin Dane <notifications@github.com>:

File: app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -51,4 +51,26 @@
     <item name="android:textSize">18sp</item>
   </style>
 
+  <style name="CardView.Default" parent="CardView">
```
Comment (tasomaniac) [inline app/src/main/res/values/styles.xml]: Can be named as Widget.GDG.CardView

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -51,4 +51,26 @@
...
+    <item name="android:layout_margin">@dimen/keyline_1_minus_8dp</item>
+    <item name="cardCornerRadius">2dp</item>
+    <item name="cardElevation">4dp</item>
+    <item name="cardUseCompatPadding">true</item>
+  </style>
+  <style name="CardView.GDE" parent="CardView.Default">
```
Comment (tasomaniac) [inline app/src/main/res/values/styles.xml]: And these can be named like Widget.GDG.CardView.GDE without using parent.

File: app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java (hunk 1 above)
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/about/ContributorAdapter.java]: It is wrong to use View.inflate here anyway. We should use layoutinflater

**Comment by tasomaniac** [comment]
Wow. This is a nice refactor. it touches almost every screen in the app. Can you add little bit more screenshot into description?

**Comment by friedger** [comment]
I didn't manage to make more screenshots, sorry.

I hope to have it in alpha so everyone can fix the issues..

**Comment by tasomaniac** [comment]
Please don't merge this or yet. I found some visual issues like gde list have top padding that crops the GDE items. 
And I think there is two much padding on the phone. 

I am mobile with very bad Internet connection. I won't be able to send a detailed response for now.

**Comment by friedger** [comment]
I won't do anything on this for the next days. Feel free to take over.

**Comment by tasomaniac** [comment]
It is sad that this is closed. I tried to rebase it but there were too many
conflicts and I gave up. Are you planning to do it again?

**Comment by friedger** [comment]
I'll cherry pick later.

**Pull request closed without merging**